packages/hyperion-autologging/src/ALNetworkPublisher.ts

```diff
@@ -159,14 +166,20 @@ function captureFetch(options: InitOptions): void {
         eventTimestamp: performanceAbsoluteNow(),
         eventIndex: ALEventIndex.getNextEventIndex(),
         flowlet,
+        triggerFlowlet: flowlet?.data.triggerFlowlet,
         ...request,
         metadata: {},
       });
     } else {
       ephemeralRequestEvent = null;
     }
 
+    const parentTriggerFlowlet = flowletManager.top()?.data.triggerFlowlet;
+    const triggerFlowlet = new flowletManager.flowletCtor(`fetch(method:${request.method},url:${request.url})`, parentTriggerFlowlet);
```

Comment:
Good point! I actually left this here hesitantly. I wanted to also test how it will work with graphapi marking. 
If we decided to keep it, certainly only pathname makes more sense.